server/utils/s3.js

```diff
@@ -21,25 +21,23 @@ function uploadFile(file) {
     }
     return s3.upload(uploadParams).promise();
 }
-exports.uploadFile = uploadFile;
+;
 
-// download a file from s3
-function getFileStream(fileKey) {
+export function getFile(fileKey) {
     const downloadParams = {
         Key: fileKey,
         Bucket: bucketName
     }
     return new Promise((resolve, reject) => {
         s3.getObject(downloadParams, (err, data) => {
             if (err) {
-                reject({message: "Access Denied: Invalid key for image"})
+                reject(err.message)
```

Comment:
Just a minor correction here, since `await` will throw the rejected value, it might be better for the rejected value to be the entire `err` object rather than just the message in order to provide as much context as possible - this would be a better practice for testing/debugging. My bad for missing this earlier 😢